compiler/codegen/library.cpp

```diff
@@ -350,6 +350,8 @@ static void makePYXFile(std::vector<FnSymbol*> functions) {
 
     gGenInfo->cfile = pyx.fptr;
 
+    fprintf(pyx.fptr, "#!python\n");
```

Comment:
My understanding is that this is a python comment-directive, not a bash level construct

Reply:
But I could be entirely mistaken, I grabbed this from a StackOverflow post

Reply:
[The response in question](https://stackoverflow.com/a/47704932/7490887)

Reply:
Oh, I guess it doesn't matter if it is in python or bash

Reply:
Ah, and here it is straight from the source: https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#globally

Nevermind then!

Reply:
But so then I'm confused.  If it's in python3, I don't understand why it would need to set the language_level (it seems weird that the language_level directive wouldn't match the version being used)